### PR TITLE
Add package planetaryimge

### PR DESCRIPTION
**Pull request by pbvarga1**

### CODE REVIEW

File: recipes/planetaryimage/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```
Comment (jakirkham) [inline recipes/planetaryimage/meta.yaml]: Is it linking to `numpy`? For example, does it have Cython code or C/C++ code that makes use of NumPy? If not, `numpy` can be dropped from `build` and the `x.x` can be dropped from `run`.

Comment (pbvarga1) [inline recipes/planetaryimage/meta.yaml]: Ahhh ok got it thanks

**Comment by jakirkham** [comment]
One easy comment. Otherwise LGTM.

**Comment by jakirkham** [comment]
Thanks @pbvarga1.

**Pull request merged**